tests/integration_python/pixi_build/test-data/rattler-build-backend/smokey/pixi.toml

```diff
@@ -0,0 +1,13 @@
+[project]
+authors = ["Wolf Vollprecht <w.vollprecht@gmail.com>"]
+channels = ["conda-forge"]
+description = "Add a short description here"
+name = "smokey-build"
+platforms = ["osx-arm64", "linux-64", "osx-64", "win-64"]
+preview = ["pixi-build"]
+version = "0.1.0"
+
+[tasks]
+
+[dependencies]
+smokey = { path = "../recipes/smokey" }
```

Comment:
I think it would be clearer if you would specify the exact file. Because this might also be a pixi.toml

Reply:
With how things work right now that isn't possible actually.